src/core/templates/security/register_user.html

```diff
@@ -15,16 +15,27 @@ <h2>
 {#  <div class="page-header">#}
 
 {% block instructions %}
-  <strong>
-    Please enter your details to register for an account.
-  </strong>
+  <p>
+    <strong>
+      Please enter your details to register for an account.
+    </strong>
+  </p>
+  <p>
+    <strong>
+      If you already have an account, you may
+      <a href="{{ url_for_security('login') }}">log in</a> here.
+    </strong>
+  </p>
 {% endblock %}
 
 {% block form_body %}
   <form action="{{ url_for_security('register') }}" method="POST" name="register_user_form">
     {{ register_user_form.hidden_tag() }}
 
     {{ render_field_with_errors(register_user_form.username) }}
```

Comment:
I love jinja2 -- being able to use normal function calling syntax in templates is something I miss.